test/chartbasicsfactory_test.js

```diff
@@ -58,7 +60,7 @@ describe('BasicsChart', function() {
       TestUtils.renderIntoDocument(elem);
     }
     catch(e) {
-      expect(console.error.callCount).to.equal(7);
+      expect(console.error.callCount).to.equal(9);
```

Comment:
Correct - we're not writing such tests in the new code in viz. The problem is that React will only show each error message once, so you end up with cascade issues in your tests.

Suggestions about what to do about the older codebases? Options I see are:
- keep adding these tests
- just don't write new tests like this
- don't write new tests like this *and* delete them if you feel like it in test files you touch, especially if they break
- don't write new tests like this *and* actively search out and delete them in test files you touch

Looping in @krystophv as well